lib/node-arguments.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const minimist = require('minimist');
+const omit = require('lodash/omit');
+
+/**
+ * @param {string[]} confParams
+ * @param {string} cliParams
+ * @return {object}
+ */
+function normalizeNodeArguments(confParams, cliParams) {
+	return omit({
+		...minimist(confParams || []),
+		...minimist(cliParams.split(' '))
```

Comment:
Are there Node.js arguments that take strings? Cause then splitting on the empty string won't work.